weather_mv/loader_pipeline/pipeline.py

```diff
@@ -41,12 +51,73 @@ def pattern_to_uris(match_pattern: str) -> t.Iterable[str]:
         yield from [x.path for x in match.metadata_list]
 
 
+def validate_region(output_table: str, temp_location: t.Optional[str] = None, region: t.Optional[str] = None) -> None:
+    """Validates non-compatible regions scenarios by performing sanity check."""
+
+    def __do_cleanup(bigquery_client: bigquery.Client, storage_client: storage.Client,
```

Comment:
nit: can we change this name to `cleanup` or `_cleanup`?

Reply:
Changed to `__cleanup`.